packages/cli/src/commands/upgrade.js

```diff
@@ -131,6 +132,25 @@ const runUpgrade = ({ d, tag }) => {
   ]
 }
 
+const SEMVER_REGEX = /(?<=^v?|\sv?)(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)(?:-(?:0|[1-9]\d*|[\da-z-]*[a-z-][\da-z-]*)(?:\.(?:0|[1-9]\d*|[\da-z-]*[a-z-][\da-z-]*))*)?(?:\+[\da-z-]+(?:\.[\da-z-]+)*)?(?=$|\s)/gi
+const validateTag = (tag) => {
+  const isTagValid = tag === 'rc' || tag === 'canary' || tag === 'latest' || SEMVER_REGEX.test(tag)
```

Comment:
```suggestion
  const isTagValid =
    tag === 'rc' ||
    tag === 'canary' ||
    tag === 'latest' ||
    SEMVER_REGEX.test(tag)
```

fix linter complaint